test/unit/helpers/admin/tabbed_nav_helper_test.rb

```diff
@@ -1,7 +1,8 @@
 require "test_helper"
 
 class TabbedNavHelperTest < ActionView::TestCase
-  test "#secondary_navigation_tabs_items for draft edition edit page" do
+  test "#secondary_navigation_tabs_items for draft edition edit page when user has editor access" do
```

Comment:
We should test the different situations explicitly, rather than use the generic "has (no) editor access":
- user is a govuk editor
- user is a welsh editor and the edition is welsh
- user is a welsh editor and the edition is not welsh